### PR TITLE
watsonx.ai L4 lab - Introduction to Generative AI

### PR DESCRIPTION
**Pull request by maurelya**
Fixes https://github.com/ibm-build-lab/VAD-VAR-Workshop/issues/316

Content changes:

- [Any content updates here](http://localhost:8000/en/watsonx/watsonxai/201)
- This is lab 201
- Changed lab `Large language model application building blocks` to lab 202

Code changes:

- Any code changes here

### CODE REVIEW

File: content/Watsonx/WatsonxAI/201.md [hunk 1]
```diff
@@ -1,276 +1,650 @@
...
+    It’s important to understand the following facts about tokens:
 
-Classification is one of the most established use cases in Natural Language Processing (NLP). Using LLMs for classification has pros and cons. Consider the following when using LLMs for classification:
+    - All LLMs have a limit for the number of supported tokens. The maximum number of
+        tokens is usually captured in documentation or in the UI, as you’ve seen in the **Prompt Lab.**
+    - The maximum number of tokens includes both input and output tokens. This means that you can’t provide an unlimited number of examples in the prompt. In addition to that, each model has the maximum number of output tokens (see [documentation](https://dataplatform.cloud.ibm.com/docs/content/wsj/analyze-data/fm-models.html?context=wx&audience=wdp)).
+    - Some vendors have daily/monthly token limits for different plans, which should be considered when selecting an LLM platform.
 
-<QuizAlert text='This will be on the quiz' />
+    Example of token limits (from documentation):
```
Comment (r2raman) [inline content/Watsonx/WatsonxAI/201.md:398]: Keep 1 line gap between the last bullet and the start of the paragraph.

File: content/Watsonx/WatsonxAI/201.md [hunk 2]
```diff
@@ -1,276 +1,650 @@
...
+    In our experience of working with clients, some of the top generative AI use cases are:
+    - Content summarization
+    - Content classification
+    - Content generation
+    - Content extraction, NER (Named Entity Recognition)
+    - Sentiment analysis
+    - Question answering with RAG
 
-   - LLMs have a limit for the number of supported tokens. The maximum number of tokens is usually captured in documentation or in the UI, as you’ve seen in the **Prompt Lab**. 
+    Now that you have reviewed and created prompts, we will test the integration of LLMs with
```
Comment (r2raman) [inline content/Watsonx/WatsonxAI/201.md:434]: Keep 1 line gap between the last bullet and the start of the paragraph.

**Review by r2raman — CHANGES_REQUESTED** [review]
Just checking if we can use jupiter hub to execute the python code.

**Comment by maurelya** [comment]
> Just checking if we can use jupiter hub to execute the python code.

The Python scripts use the Streamlit library to open a page in the web browser.
It’s not currently possible to run Streamlit inside Jupyter. https://discuss.streamlit.io/t/running-streamlit-in-a-jupyter-cell/584

**Review by r2raman — APPROVED** [review]

**Pull request merged**